### PR TITLE
Added automation example for qrcode integration

### PR DESCRIPTION
**Pull request by stoffus**
## Proposed change

Added an automation example to the qrcode integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @stoffus, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#17677).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/qrcode.markdown [hunk 1]
```diff
@@ -39,3 +39,21 @@ source:
...
+
+An automation using this integration could look like this:
+
+```yaml
+automation:
+  - alias: 'Catch QR code and arm the alarm system'
+    trigger:
+      platform: state
+      entity_id: image_processing.qr_front_door
+      to: !secret ARM_QR_CODE_VALUE
```
Comment (frenck) [inline source/_integrations/qrcode.markdown]: Please don't use secrets in examples

File: source/_integrations/qrcode.markdown [hunk 2]
```diff
@@ -39,3 +39,21 @@ source:
...
+    action:
+      service: alarm_control_panel.alarm_arm_away
+      entity_id: alarm_control_panel.home_alarm
+      data:
+        code: !secret MY_ALARM_CODE
```
Comment (frenck) [inline source/_integrations/qrcode.markdown]: Please don't use secrets in these examples

File: source/_integrations/qrcode.markdown [hunk 3]
```diff
@@ -39,3 +39,21 @@ source:
...
       type: string
 {% endconfiguration %}
+
+## Basic example
+
+An automation using this integration could look like this:
+
+```yaml
+automation:
+  - alias: 'Catch QR code and arm the alarm system'
```
Comment (frenck) [inline source/_integrations/qrcode.markdown]:
```suggestion
  - alias: "Catch QR code and arm the alarm system"
```

File: source/_integrations/qrcode.markdown [hunk 4]
```diff
@@ -39,3 +39,21 @@ source:
...
+    trigger:
+      platform: state
```
Comment (frenck) [inline source/_integrations/qrcode.markdown]: This needs to be a list, see our YAML style guide: <https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

```suggestion
      - platform: state
```

File: source/_integrations/qrcode.markdown [hunk 5]
```diff
@@ -39,3 +39,21 @@ source:
...
+      entity_id: image_processing.qr_front_door
+      to: !secret ARM_QR_CODE_VALUE
+    action:
+      service: alarm_control_panel.alarm_arm_away
```
Comment (frenck) [inline source/_integrations/qrcode.markdown]: This needs to be a list, see our YAML style guide: <https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

```suggestion
      - service: alarm_control_panel.alarm_arm_away
```

File: source/_integrations/qrcode.markdown [hunk 6]
```diff
@@ -39,3 +39,21 @@ source:
...
+      entity_id: alarm_control_panel.home_alarm
```
Comment (frenck) [inline source/_integrations/qrcode.markdown]: Please use service targets, see our style guide for more information: <https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

```suggestion
      target:
        entity_id: alarm_control_panel.home_alarm
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @stoffus 👍

**Pull request merged**